examples/eks_test_fixture/main.tf

```diff
@@ -39,17 +39,26 @@ locals {
   #   },
   # ]
 
-  worker_groups = [
+  launch_configuration_worker_groups = [
```

Comment:
Can we keep this naming consistent also? i.e. keep this as `worker_groups` and name the new one `worker_groups_launch_template`.